ayota/ddl_nlp
# Storage fix: resolves Issue #7

This resolves #7 

Because we want to be able to run our model on k-folds we had to build a methodology to not only split our dataset up into several files but store them in some reasonable fashion.

This PR re-writes the word2vec.py file.  It builds a wrapper around the original run_model function that loops through the various folds that have been previously generated and runs the model build k number of times.

The wrapper is named run_model and the original function is now named run_model_fold.  

So now when a user puts in the data they want to run over run_model it runs all folds and builts a model file for each fold.

Expected response:

The expectation is that the script pulls all fold data (train only) form a directory structure that looks like the following:

data
--term
----1
------train
--------train.txt
------test
--------test.txt
----2
------train
--------train.txt
------test
--------test.txt

The resulting model files should end up in a folder structure like the following

model
--term
----1
--------term.model
----2
--------term.model

Directions:
1. Run wikipedia_ingest.py on some term 'fox'
2. Create a directory 'fox' under the 'ontology' directory and put a text file under that folder.  This will act as the ontology generated file(s). In fact make 2 fake files.
3. Run the generate_folds.py on the term 'fox'
4. Please run the word2vec.py script to make sure it is working as described above and that all of the files nad folders and generated properly.
